client/state/sites/selectors.js

```diff
@@ -123,3 +123,46 @@ export function getSiteByUrl( state, url ) {
 
 	return site;
 }
+
+/**
+ * Returns a site's plan object by site ID.
+ *
+ * The difference between this selector and sites/plans/getPlansBySite is that the latter selectors works
+ * with the /sites/$site/plans endpoint while the former selectors works with /sites/$site endpoint.
+ * Query these endpoints to see if you need the first or the second one.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {?Object}        Site's plan object
+ */
+export function getSitePlan( state, siteId ) {
+	const site = getSite( state, siteId );
+
+	if ( ! site ) {
+		return null;
+	}
+
+	return site.plan;
+}
+
+/**
+ * Returns true if site is currently subscribed to supplied plan and false otherwise.
+ *
+ * @param  {Object}   state         Global state tree
+ * @param  {Number}   siteId        Site ID
+ * @param  {Number}   planProductId Plan product_id
+ * @return {?Boolean}               Whether site's plan matches supplied plan
+ */
+export function isCurrentSitePlan( state, siteId, planProductId ) {
+	if ( planProductId === undefined ) {
```

Comment:
Lately, I was reviewing code which checked for:

```js
if ( plan.raw_price ) { ... }
```
However, as `raw_price` can be also `0` (free plan) and `0` == `false`, this condition didn't work for all the times (for the free plan). 

On the other hand, currently, there is no plan product ID less than `1` so this condition is not protecting against anything. I still think it should be left there. Or?